### ISSUE TITLE
Investigate writing a custom Git merge driver for JSON files

### ISSUE THREAD

**Issue by maltem-za** [forTEXT/catma#351]
See https://github.com/forTEXT/catma-admin-utils/commit/1bd0c4f1ff3d1791bea2253107d3778d4c02d2c4

**Comment by maltem-za** [comment]
@mpetris Did you ever consider doing something like this?

**Comment by mpetris** [comment]
No, not as a custom driver. When we still had the submodules in place we did custom conflict resolution with merging outside of git and marked the conflict solved then. But that was mostly with the submodules housekeeping files, which are in a format of their own. Our JSON might be more predictable. The references you mentioned sound promising.

**Comment by maltem-za** [comment]
@mpetris Thanks, was just wondering if it's something you maybe already looked into in the past, before I go down that rabbit hole ;)
As you said though, with how it worked before it was of course a different challenge.
